Engine/Main/Main.cpp

```diff
@@ -28,6 +28,7 @@ int main(int argc, char ** argv)
 
 	// Random number generator
 	pcg32_srandom(time(NULL), 0);
+	srand(time(NULL));
```

Comment:
if you need a random generator pcg32_srandom is not enough?

Reply:
mario told me to put it there, it's because i use rand() on the particle system and therefore to have a seed or something like that